README.md

```diff
@@ -47,6 +47,18 @@ Docker should function normally, with the following caveats:
 
 * Additional certificates used by the Docker daemon to authenticate with registries need to be located in `/var/snap/docker/common/etc/certs.d` instead of `/etc/docker/certs.d`.
 
+## NVIDIA support
```

Comment:
We haven't yet make it work on Classic in expected ways (using hostfs libraries), thus maybe we should ommit this from README.md for now, or like mention that it's targeting Ubuntu Core 22 only for now.

Reply:
Or maybe nobody reads this anyway =)